### PR TITLE
add notes for rancher extra binds

### PR DESCRIPTION
**Pull request by cofyc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
/kind documentation
> /kind test
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/107#issuecomment-521478232

I was asked several times about how to use local volume provisioner in Rancher. I think we should explicitly document it.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```

```

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -195,3 +195,8 @@ them to be discovered again.
 Another solution is to add a bind in the kubelet deployment configuration to
 mount the parent directory of local volumes into the kubelet container at the
 same path. This requires restarting the kubelet container.
+
+A famous Kubernetes distribution that runs the kubelet in a container is
```
Comment (msau42) [inline docs/faqs.md:199]: simplify the first part of the sentance to something like:

"For Rancher clusters, users must configure ..."

Comment (cofyc) [reply docs/faqs.md]: updated

**Pull request merged**